README.md

```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
 const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
+
+
+**CF Notes** - based on 11/30 convo do we want to include specifics on:
+
+-  Are there character limits for error messages?
```

Comment:
I don't know.  Want to try deploying a sheet with a reallllly long message and see where it fails.  test around 256 (smaller larger), 512, and 1024.  The backend might save the message and the frontend might do something wonky on display.

Reply:
this is more of a presentation issue on the front end, removing from FAQ.